HelpMyStreetFE/HelpMyStreetFE/ViewComponents/ViewLocationPopupViewComponent.cs

```diff
@@ -68,7 +72,7 @@ public async Task<IViewComponentResult> InvokeAsync(int? jobId, int? requestId,
                     Coordinates = null,
                     Distance = null,
                     PostCode = null,
-                    encodedJobID = Base64Utils.Base64Encode(jobSummary.JobID)
+                    encodedJobID = Base64Utils.Base64Encode(jobId.HasValue ? jobId.Value : requestId.Value)
```

Comment:
Is this OK with a RequestID?  Looks like it'll flow through to get-directions-link which expects a JobID